plugins/modules/ec2_import_image.py

```diff
@@ -498,7 +497,7 @@ def main():
     state = module.params.get("state")
 
     try:
-        client = module.client("ec2", retry_decorator=AWSRetry.jittered_backoff())
+        client = module.client("ec2")
```

Comment:
I think we can remove this try/except block per the discussion [here](https://github.com/ansible-collections/amazon.aws/pull/2138#discussion_r1660951340).